src/__tests__/Dialog.tsx

```diff
@@ -10,7 +9,7 @@ expect.extend(toHaveNoViolations)
 const comp = (
   <Dialog isOpen onDismiss={() => null} aria-labelledby="header">
     <Dialog.Header id="header">Title</Dialog.Header>
-    <Box p={3}>
+    <Box sx={{p: 3}}>
```

Comment:
We don't need to make these changes to Box since it's the only component that still accepts system props.